test/core/ticktree/TickTreeCleanOffer.t.sol

```diff
@@ -6,42 +6,42 @@ import {TickTreeTest, TestTickTree} from "./TickTreeTest.t.sol";
 import "mgv_src/MgvLib.sol";
 import "mgv_lib/Debug.sol";
 
-// Tests of Mangrove.cleanByImpersonation's interaction with the tick tree.
+// Tests of Mangrove.cleanByImpersonation's interaction with the bin tree.
 //
 // The tests use the following pattern:
-// 1. we establish a Mangrove tick tree where there may be offers at:
-//   - the offer to be cleaned's tick (including the offer itself)
+// 1. we establish a Mangrove bin tree where there may be offers at:
+//   - the offer to be cleaned's bin (including the offer itself)
 //   - a higher tick
 //   - a lower tick
```

Comment:
```suggestion
//   - a higher bin
//   - a lower bin
```

Reply:
fixed in 5ff12ac03bae78f540e71b471e3e655475c46eb7